omero/developers/whatsnew.txt

```diff
@@ -23,6 +23,28 @@ by :javadoc:`Chmod2
 Java Gateway
 ^^^^^^^^^^^^
 
-OMERO model
+OMERO Model
 ^^^^^^^^^^^
 
+-  Added ``Folder``, ``FolderImageLink`` and ``FolderRoiLink``
+-  Added ``ProjectionDef`` to ``RenderingDef``
+-  Added "Arrow" ``markerStart`` and ``markerEnd`` to ``Line`` and
+   ``Polyline``
+-  Added `ReverseIntensityMapContext` for ROMIO
```

Comment:
This is a specific type of CodomainMapContext so I don't think it is necessary to list it